types/index.d.ts

```diff
@@ -426,6 +426,14 @@ declare module '@datafoldcom/react-base-table' {
      * Number of rows to render above/below the visible bounds of the list
      */
     overscanRowCount?: number;
+    /**
+    * Number of rows to render above/below the visible bounds of the list
```

Comment:
rows or columns?

Reply:
Right, that was wrong copy-pasting. Will fix now